scorecard: Fix basic tests run when 'olm-deployed: true' (#2274)
operator-framework/operator-sdk: CHANGELOG.md

### Bug Fixes
- Fix issue faced in the Ansible based operators when `jmespath` queries are used because it was not installed. ([#2252](https://github.com/operator-framework/operator-sdk/pull/2252))

> - Fix scorecard basic test when run with `olm-deployed` option set to true ([#2274](https://github.com/operator-framework/operator-sdk/pull/2274))

Reviewer: ```suggestion
- Fix error `unrecognized install strategy` which is shown when the scorecard basic test is configured with `olm-deployed: true`. ([#2274](https://github.com/operator-framework/operator-sdk/pull/2274))
```

wdyt?

Reviewer 2: Something more like:
```
Fix scorecard behavior such that a CSV file is read correctly when `olm-deployed` is set to `true`.
```

Author: any preference ? I prefer estroz's one. Shall I commit that ?

Reviewer: It can be the @estroz's one. 👍

Author: Fixed
